src/main/java/scrumweb/project/service/ProjectService.java

```diff
@@ -74,4 +72,16 @@ public ProjectMemberDto addMember(ProjectMemberDto projectMemberDto){
                 .map(type -> new IssueType(type, project))
                 .collect(Collectors.toSet());
     }
+
+    public List<ProjectDto> displayAllUserProject(){
```

Comment:
hmm... ja bym tu zrobił tak, że funkcja w argumencie będzie przyjmowała UserAccount i potem return userAccount.getProjects.stream()
.map(project>projectAsm.createProjectDto(project)
.collect(Collectors.toList());
a w kontrolerze securityContextService.getCurrentUserAccount()
wtedy będziemy mieli jedną funkcję dla pobierania wszystkich projektów użytkownika i nie tylko zalogowanego, a jakiegokolwiek ;)

Reply:
dodatkowo jeszcze: zauważyłem, że mamy relację UserProfile do Project, trzeba było by zmienić to. UserAccount musi mieć projekty :)
tak nie bardzo ja pomyślałem i źlę poszło z tym profilem ;(

Reply:
i jeszcze zmień nazwę tej funkcji np. na getAllProjects

Reply:
również zapomniałeś o testach